__tests__/api.test.js

```diff
@@ -40,4 +40,25 @@ describe("News API BACKEND PROJECT", () => {
             })
         })
     })
+    describe("GET /api that returns documentation of available api endpoints", () => {
+
+        test('returns a 200 response status', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
+        })
+        test('returns a documentation with the expected properties', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
```

Comment:
Nice, but you are testing the status code on both of these tests, so either the first test can be removed or the second test should not have the .expect(200)